### ISSUE TITLE
c_cxx fns_candy_style_transfer does not compile with newest onnxruntime

### ISSUE THREAD

**Issue by lifa08** [microsoft/onnxruntime-inference-examples#54]
When use the onnxruntime compiled from source code with latest change, `error: 'EXHAUSTIVE' undeclared` occurs.
Looks like the `EXHAUSTIVE` has been replaced by `OrtCudnnConvAlgoSearchExhaustive` in [this commit](https://github.com/microsoft/onnxruntime/commit/b876e5675badf515bb9a3948add3b7010a14c3c0), in this case, how to support different versions of onnxruntime?

**Comment by snnn** [comment]
Right. onnxruntime has binary backward compatibility(if you don't use CUDA), but source code level  compatibility  is not supported.  So, in short, I don't know how.
